.changes/resize-false-undecorated-edge.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri-runtime-wry": "patch:bug"
+---
+
+Fix window edge not working after setting resizable false and decorated false dynamically
```

Comment:
I am not sure I understand the problem, if decoration is false and resizable is false, window edge should not work

Reply:
Shouldn't the edges of an undercoated window with resize disabled receive events and being clickable?

Reply:
yeah it should, is it not receiving these events?

Reply:
yep

Reply:
ah I see where the bug is, I will push a better fix

Reply:
@Legend-Master could you give 14af33bc765751b07de187884995c8066902a697 a test?

Reply:
Doesn't seem to work, after setting both resize and decoration to false, the entire window becomes not interactable

Reply:
I retested and seemed like I can click just fine.

Reply:
nvm, I was mislead by the hover effect, I reverted my changes

Reply:
kinda weird that `HTTRANSPARENT` didn't work, will need to research it more I guess. Your solution is fine for now.